src/layouts/SingleColumn.scss

```diff
@@ -0,0 +1,11 @@
+.singleColumnLayout {
+  height: 100%;
+
+  padding: 0 1em;
+  margin: 0 auto;
+
+  max-width: 550px;
```

Comment:
any reason for this arbitrary width?

Reply:
https://www.figma.com/file/WSXjCGc1k6FVI093qhlzOP/04-Recieving-share-link?node-id=221%3A10009

That's more or less the width specified by the designs. Any suggestions for a nicer generalisation?